spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,33 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'returns https success' do
+      user = User.create(name: 'Austin', id: '1')
+      get user_post_path(user)
+      expect(response).to have_http_status(:success)
+    end
+  end
+  describe 'GET /show' do
+    it 'returns https success' do
+      user = User.create(name: 'Austin', id: '1')
+      post = Post.create(author: user, title: 'Hi', text: 'First Post')
+      get user_post_path(user, post)
+      expect(response).to have_http_status(:success)
+    end
+  end
+  describe 'Elements for posts' do
+    it 'render the template of post index' do
+      user = User.create(name: 'Austin', id: '1')
+      get user_posts_path(user)
+      expect(response.body).to render_template('index')
+      expect(response.body).to include('<h1>Show all users</h1>')
+    end
```

Comment:
- [ ] Kindly organize your specs. This it-block is for the index posts page and so should be under the `GET /index` describe block above.